packages/core/src/feature-service-registry.ts

```diff
@@ -109,6 +109,20 @@ export interface FeatureServiceRegistryOptions {
    * registered.
    */
   readonly configs?: FeatureServiceConfigs;
+
+  /**
+   * When the {@link FeatureAppManager} is configured with a
+   * {@link FeatureAppManager#moduleLoader}, to load Feature Apps from a remote
+   * location that also provide their own Feature Services, i.e. the Feature
+   * Services are included in a different bundle than the integrator bundle, it
+   * might make sense to validate external dependencies that are required by
+   * those Feature Services against the shared dependencies that are provided by
+   * the integrator. This makes it possible that an error is already thrown when
```

Comment:
This makes it possible to detect this incompatibility when registering a Feature Service with incompatible external dependencies.

Reply:
I would prefer a similar sentence to the one we decided on for the `FeatureAppManagerOptions`:

> This makes it possible that an error is already thrown when registering a Feature Service with incompatible external dependencies, and thus enables early feedback as to whether a Feature Service is compatible with the integration environment.